allennlp_models/generation/models/copynet_seq2seq.py

```diff
@@ -336,9 +336,13 @@ def _decoder_step(
         # shape: (group_size, decoder_input_dim)
         projected_decoder_input = self._input_projection_layer(decoder_input)
 
-        state["decoder_hidden"], state["decoder_context"] = self._decoder_cell(
-            projected_decoder_input, (state["decoder_hidden"], state["decoder_context"])
-        )
+        # TODO (epwalsh): remove the autocast(False) once torch's AMP is working for RNNs.
```

Comment:
Maybe make this more specific?  Same with the other files.
```suggestion
        # TODO (epwalsh): remove the autocast(False) once torch's AMP is working for LSTMCells.
```